torch/ao/quantization/observer.py

```diff
@@ -1204,7 +1212,8 @@ def reset_histogram(self, x: torch.Tensor, min_val: torch.Tensor, max_val: torch
             x, self.bins, min=min_val, max=max_val, out=self.histogram  # type: ignore[arg-type]
         )
 
-    def forward(self, x_orig: torch.Tensor) -> torch.Tensor:
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x_orig = x
         if x_orig.numel() == 0:
             return x_orig
         x = x_orig.detach()
```

Comment:
we can rename this to `x_detach` to reduce confusion I think

Reply:
This makes sense if we keep 'x' as the input. It does clutter the code a bit. If we decide on having 'x' as input instead of 'x_orig' I would make that change, otherwise keep it as is.